@navikt/core/react/src/copybutton/CopyButton.tsx

```diff
@@ -64,7 +65,8 @@ export interface CopyButtonProps
   title?: string;
   /**
    * Accessible label for icon in active-state (ignored if text is set).
-   * @default "Kopiert"
+   * @default "Kopiert!"
+   * @deprecated Use `activeText` instead
    */
   activeTitle?: string;
```

Comment:
Since no-one uses this, can it be removed completely?
https://metabase.ansatt.nav.no/question/2711-radata-fra-component-usage?component=CopyButton&prop=activeTitle&major=